src/test/java/xdev/ui/tree/XdevTreeNodeTest.java

```diff
@@ -51,6 +59,23 @@ private XdevTreeNode createTestNodes(){
 	@Test
 	public void getSubTree(){
 		
-		this.createTestNodes().getSubTree();
+		List<TreeNode> nodes = this.createTestNodes().getSubTree();
+		
+		XdevTreeNode node1 = (XdevTreeNode) nodes.get(0);
+		XdevTreeNode node2 = (XdevTreeNode) nodes.get(1);
+		
+		Assert.assertEquals("root", node1.getCaption());
+		Assert.assertEquals("level1", node2.getCaption());
 	}
+	
+	
+	@Test
+	public void searchTreeNodes(){
```

Comment:
This doesn't look like our "normal" formatter

Reply:
Fixed :)